changelog.md

```diff
@@ -6,6 +6,15 @@ The **minimum recommended version is [2.16.2](#2.16.2)**.
 Because version 3 of the Azure Cosmos DB .NET SDK includes updated features and improved performance, we’ll retire version 2.x of this SDK on 31 August 2024.  As a result, you’ll need to update your SDK to version 3 by that date. We recommend following the [instructions](https://docs.microsoft.com/azure/cosmos-db/sql/migrate-dotnet-v3?tabs=dotnet-v3) to migrate to Azure Cosmos DB .NET SDK version 3.
 
 ## Release notes
+### <a name="2.17.0"></a> 2.17.0
+* Fixed request diagnostics to reset thread starvation flag once starvation is detected
+* Fixed request diagnostics to correctly include split responses
+* Fixed session token when the target partition was not in the local cache, the Global Session Token would be sent and could cause BadRequest with large header errors
+* Improved TCP performance by reducing allocated buffer size for requests
+* Improved Service Unavailable errors to include substatus codes for different known scenarios for easier troubleshooting
+* Improved request diagnostics to include service endpoint information
+* Improved client availability by enabling the account information refresh upon client creation, which helps detect regional changes without a required failure happening
```

Comment:
Improved availability by avoiding retries on replicas that previously failed for that request
Improved availability from avoiding replica during cache refreshes
Fixed issue where if cancellation token was equal to or less than request timeout the address cache would not get updated

Reply:
Fixed a bug where SDK would use preferred regions even if EndpointDiscovery is disabled
Improved availability by always initializing background refresh of account information along with the client